.rubocop_todo.yml

```diff
@@ -13,40 +13,33 @@ Lint/AmbiguousRegexpLiteral:
     - 'test/vagrant-service-manager/binary_handlers/adb_openshift_binary_handler_test.rb'
 
 # Offense count: 1
-Lint/HandleExceptions:
+Lint/Eval:
   Exclude:
-    - 'lib/vagrant-service-manager/service.rb'
+    - 'features/support/env.rb'
 
-# Offense count: 13
+# Offense count: 17
```

Comment:
Might be worse to do all this rubocop fiddling separately. It is not really related to the issue at hand at all.